### DISCUSSION TITLE
[Run issue]: Scoring a whole patient organ in CT/CBCT dosimetry

### DISCUSSION THREAD

**Discussion by ninamcw** [OpenTOPAS/OpenTOPAS#93] [Run-Problem]
### Contact Details

nina.mcwilliams@ucdconnect.ie

### What was the issue?

Hi all,

I'm using a patient CT DICOM dataset for the simulation of a diagnostic CT and CBCT scanner. At the moment, I am using the Schneider material conversion for the dataset when I pull it into the simulation. I have a question with regards to the scoring of an organ in this simulation. I would like to score to dose deposition in a whole organ within the patient abdomen i.e. the kidneys. Photo attached as an example of the segmentation. Considering as the organ is made up of several voxels spread across multiple CT slices, I'm not sure if I am tackling the issue correctly and could do with some guidance.

1) I have seen the RT Scoring grid example and the RT dose structure format which allows you to only specify the region you want to score - _sv:Sc/EnergyFluence/OnlyIncludeifinRTStructure_. But I wondered if this was the right path for scoring the whole volume of an organ? However as this is a diagnostic CT, I do not have an RT DICOM file. 

2) Secondly I had thought maybe I could tackle it similar to that of the XCAT phantom by defining/setting the material type of that organ, ideally based on the HU values. Ideally, that way I would be able to adjust the HU of the kidneys and repeat the simulation quantifying the dose to the organ?

Has anyone tackled anything similar? Open to any suggestions and I really appreciate the help.

Best wishes,
Nina

![Kidney_Slicer](https://github.com/user-attachments/assets/2775bc94-3deb-439d-a8f5-9edd8d386667)

### TOPAS category

Scoring

### Version

3.9

### What Operative System (OS) are you running on?

Linux

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Attestation

- [X] I testify that I have fully read the bug reporting guideline beforehand.

**Comment by JoseRamosMendez** [comment]
Hi there,
1. Would be ideal. But as you stated, there's no DICOM-RT 
2. Override the HU is a good idea, but DICOM-CT has HU of similar value in other places, so you'll get voxels outside the kidney region. 
  2.a. You could export the segmented image as a binary file, with the body contour also included. This way you can set the geometry in a similar way as the XCAT example. Alternatively:
  2.b. You could get the dose in the entire phantom, and then use post-processing to filter the dose. For that you can generate and use an index boolean mask matrix from the segmented image (as shown in Slicer, kidney = True, elsewhere=False).
Best.

**Comment by ninamcw** [comment]
Hi Dr Mendez,

Thanks for your helpful reply.
That was the issue I was having with the soft tissue regions and the similar HUs. I would also like to investigate the change in HU on the dosimetry of the organ so the option 2a above might be a good method to incorporate that aspect. Fo the post-processing filter, that would be a python script to filter out the tissue types that were not of interest? 

Than you for your help and advice,
Best wishes,
Nina
